File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        nums1=[]
        nums2=[]
        count=0
        for i in nums:
            nums1.append(i)
        for i in range(len(nums1)):
            if nums1[i]==0:
                nums2.append(count)
                count=0
            else:
                count+=1
        nums2.append(count)
        return max(nums2)
           
        
        

            

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int maxi=0;
        int count=0;
        int len=nums.length;
        if(len==1 && nums[0]==0) return 0;
        if(len==1 && nums[0]==1) return 1;
        for(int i:nums)
        {
            if(i==1)
            {
                count+=1;
                if(count>maxi)
                {
                    maxi=count;
                }
                
            }
            else
                {
                    count=0;
                }
        }

        return maxi ;


        
    }
}